src/content/homepage/charts/second-chart/convert-msaxlsx-to-html.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+if [ $# -ne 1 ]; then
+     echo "usage: $0 <xlsx-filename>"
+     exit 1
+fi
+
+# https://github.com/dilshod/xlsx2csv (this gracefully handles unicode nastiness, thankfully).
+pip install xlsx2csv
+xlsx2csv -d '|' $1 tmp-msa.csv
+
+echo '<select name="hmda_chart_2_msa" id="hmda_chart_2_msa">
+<option selected value="CBSA00000">U.S. Total</option>' > msa-dropdown.html
+awk -F "|" '{ if(NR>1){print("<option value=\"CBSA"$1"\">"$2"</option>")}}' tmp-msa.csv >> msa-dropdown.html
+echo '</select>' >> msa-dropdown.html
+
+echo 'successfully created msa-dropdown.html'
+
+# cleanup tmp file
+rm -f tmp-msa.csv
```

Comment:
That can go. (It forces the remove and overcomes rm -i when that is aliased to rm ... which I hope most people around here do.)